doc/rados/operations/pools.rst

```diff
@@ -724,6 +724,21 @@ For example::
 This ensures that no object in the data pool will receive I/O with fewer than
 ``min_size`` replicas.
 
+If unset, ``min_size`` defaults to ``size - size/2``. For example:
```

Comment:
@l-mb this is not accurate, when a pool is created. we set the `min_size` to `min(osd_pool_default_min_size, osd_pool_default_size)` if `osd_pool_default_min_size` is not 0. otherwise, we use `osd_pool_default_size - osd_pool_default_size / 2`.

and this might be misleading, as the `min_size` does not update itself when `size` changes. it changes only when the new `size` is smaller than the existing `min_size`, in that case, `min_size` is decreased to `size`.